dtoolkit/decorator.py

```diff
@@ -1,29 +0,0 @@
-from __future__ import annotations
-
-from functools import wraps
-
-from ._checking import check_dataframe_type
-from .util import fargs_dict
-
-
-def require_type_factory(check: callable):
-    """Transform a checking var type function to a checking decorator."""
-
-    @wraps(check)
-    def require_type(var: str):
-        def wrapper(method):
-            @wraps(method)
-            def decorator(*args, **kwargs):
-                check_args = fargs_dict(method, *args, **kwargs)
```

Comment:
there we want `method` is the original one, but `method` would change after it is wrapped by the decorator. So `fargs_dict` can't get the right parameters.